### PR TITLE
capitalize Fairlearn in dashboard

### PR DESCRIPTION
**Pull request by rihorn2**
Change capitalization of fairlearn to Fairlearn. Set starting page for view with precomputed metrics to the feature selection page.

### CODE REVIEW

File: fairlearn/widget/js/src/fairlearnDashboard.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ export class FairlearnView extends DOMWidgetView {
             predictedY={data.predicted_ys}
             trueY={data.true_y}
             modelNames={data.model_names}
+            customMetrics={[]}
```
Comment (riedgar-ms) [inline fairlearn/widget/js/src/fairlearnDashboard.tsx]: I believe we need to chat about this at some point. However it's not MVP for the AzureML integration

File: visualization/FairnessDashboard/src/FairnessWizard.tsx [hunk 2]
```diff
@@ -252,7 +252,7 @@ export class FairnessWizard extends React.PureComponent<IFairnessProps, IWizardS
                 parityMetrics: accuracyMetrics,
                 selectedParityKey: accuracyMetrics[0].key,
                 dashboardContext: FairnessWizard.buildPrecomputedFairnessContext(props),
-                activeTabKey: accuracyTabKey,
+                activeTabKey: featureBinTabKey,
```
Comment (riedgar-ms) [inline visualization/FairnessDashboard/src/FairnessWizard.tsx]: Is this the reason that the dashboard in Mark's part of the PoC always opens to the 'second' page?

**Comment by riedgar-ms** [comment]
Could we have a description, please? I think this is doing more than the PR title imples.

File: visualization/FairnessDashboard/src/FairnessWizard.tsx (hunk 2 above)
Comment (rihorn2) [reply visualization/FairnessDashboard/src/FairnessWizard.tsx]: yes. This should fix it to open to the feature page

File: fairlearn/widget/js/yarn.lock [hunk 3]
```diff
@@ -228,70 +239,70 @@
...
+  resolved "https://registry.yarnpkg.com/@uifabric/foundation/-/foundation-7.5.19.tgz#8f84c5c249b006d7b2bf820d6e828d08d7648195"
+  integrity sha512-zBOzpZy5H895r8lmmbeRTrJ8r1MpRTavcbnbGJK1rYGB4/1hcxNmdpH1/Dt96JPCMGHH0wRzCT/w5bdAN99k4Q==
+  dependencies:
+    "@uifabric/merge-styles" "^7.8.7"
+    "@uifabric/set-version" "^7.0.6"
+    "@uifabric/styling" "^7.10.19"
+    "@uifabric/utilities" "^7.14.1"
+    tslib "^1.10.0"
+
+"@uifabric/icons@^7.3.18":
```
Comment (riedgar-ms) [inline fairlearn/widget/js/yarn.lock]: These are all just general updates?

Comment (rihorn2) [reply fairlearn/widget/js/yarn.lock]: we don't pin these dependencies to an upper version, so as we publish our package again, it pulls in the latest ones

**Pull request merged**